Content/Items/Beach/Accessories.SeaglassLens.cs

```diff
@@ -20,12 +20,12 @@ public override void SafeSetDefaults()
 			Item.value = Item.sellPrice(silver: 50);
 		}
 
-		private void BonusDamage(Projectile projectile, NPC target, NPC.HitInfo hit, int damageDone)
+		private void BonusDamage(Projectile projectile, NPC target, ref NPC.HitModifiers modifiers)
 		{
 			Player owner = Main.player[projectile.owner];
 
 			if (owner == Main.LocalPlayer && Equipped(owner) && projectile.DamageType == DamageClass.Summon && Main.rand.NextBool(10))
-				target.SimpleStrikeNPC(damageDone / 2, hit.HitDirection, true);
+				modifiers.SetCrit();
```

Comment:
Can we double check this works? I recall initially using this as the first implementation but encountering issues